esmvaltool/cmorizers/obs/cmorize_obs_eppley_vgpm_modis.py

```diff
@@ -0,0 +1,138 @@
+# pylint: disable=invalid-name
+"""ESMValTool CMORizer for Eppley-VGPM-MODIS data from Oregon State University.
+
+Tier
+
+Source
+   http://orca.science.oregonstate.edu/data/1x2/monthly/eppley.r2018.m.chl.m.sst/hdf
+
+Last access
+   20190515
+
+Download and processing instructions
+   Download and unpack all files under a single directory
+   (no subdirectories with years) in ${RAWOBS}/Tier2/Eppley-VGPM-MODIS
+
+Modification history
+   20190515-A_lova_to: written.
+
+"""
+
+import logging
+import os
+import glob
+from datetime import datetime as dt
+import xarray as xr
+import numpy as np
+
+import iris
+
+from .utilities import (constant_metadata, fix_coords, fix_var_metadata,
+                        save_variable, set_global_atts)
+
+logger = logging.getLogger(__name__)
+
+
+def _fix_data(cube, var):
+    """Specific data fixes for different variables."""
+    logger.info("Fixing data ...")
+    with constant_metadata(cube):
+        if var == 'intpp':
+            cube /= 1000. * 12.01 * 86400.
+    return cube
+
+
+def extract_variable(var_info, raw_info, out_dir, attrs):
+    """Extract to all vars."""
+    var = var_info.short_name
+    cubes = iris.load(raw_info['file'])
+    rawvar = raw_info['name']
+
+    for cube in cubes:
+        if cube.var_name == rawvar:
+            fix_var_metadata(cube, var_info)
+            fix_coords(cube)
+            _fix_data(cube, var)
+            set_global_atts(cube, attrs)
+            save_variable(
+                cube,
+                var,
+                out_dir,
+                attrs,
+                local_keys=['coordinates'],
+                unlimited_dimensions=['time'],
+            )
+
+
+def merge_data(in_dir, out_dir, raw_info):
+    """Merge all data into a single file."""
+    var = raw_info['name']
+    datafile = sorted(glob.glob(in_dir + '/' + raw_info['file'] + '*.hdf'))
```

Comment:
datafile is not a very clear name for a list of files, can you improve?

Reply:
Changed `datafile` into `filelist`